e2e/files.spec.ts

```diff
@@ -114,9 +114,19 @@ const uploadFile = async (
         ? 'Upload or Link to a File'
         : 'Upload a New Version of File'
     await page.getByRole('button', { name: uploadButtonText }).click()
+
+    await expect(page.getByRole('link', { name: 'Upload File' })).toBeVisible()
+    await expect(page.getByRole('link', { name: 'Link to URL' })).toBeVisible()
+    await expect(page.getByRole('button', { name: 'Browse...' })).toBeVisible()
   })
 
   await testAuth.step('choose file', async () => {
+    // Ensure that upload type is specified before attempting upload
+    // to prevent upload error: "Unsupported external upload type specified: undefined"
+    await expect(
+      page.getByText(/all uploaded files will be stored in synapsestorage/i),
+    ).toBeVisible()
+
```

Comment:
Ensure that the upload type is specified in the upload modal before attempting upload, so that the upload type is specified -- the flakiness on the Sage larger Linux runner may be associated with this error. 

<img width="1441" alt="SWC-6556_uploadError" src="https://github.com/Sage-Bionetworks/SynapseWebClient/assets/26949006/54297b94-a0f3-49ab-b6fa-d461c54582ba">